config/core/webhooks/config-validation.yaml

```diff
@@ -24,8 +24,8 @@ webhooks:
     clientConfig:
       service:
         name: webhook
-        namespace: cloud-run-events
-    failurePolicy: Fail
+        namespace: events-system
+    failurePolicy: Ignore
```

Comment:
Is this intended to be in this PR? I actually like the change, but am not sure we want to include it in this large renaming PR.

Reply:
The reason we have it here is that we couldn't get past the webhook issue when testing without it. I can also move it to a separate PR, if preferable.